aws_lambda_builders/workflows/nodejs_npm/utils.py

```diff
@@ -48,3 +64,124 @@ def abspath(self, path):
 
     def is_windows(self):
         return platform.system().lower() == "windows"
+
+
+class DependencyUtils(object):
+
+    """
+    Collection of helper functions for managing local NPM dependencies
+    """
+
+    @staticmethod
+    def get_local_dependencies(manifest_path, osutils):
+        """
+        Helper function to extract all local dependencies in a package.json manifest
+        """
+
+        with osutils.open_file(manifest_path) as manifest_file:
+            manifest = json.loads(manifest_file.read())
+
+            if "dependencies" in manifest:
+                return dict(
+                    (k, v) for (k, v) in manifest["dependencies"].items() if DependencyUtils.is_local_dependency(v)
+                )
+            else:
```

Comment:
minor: You don't need an `else:` here, you can return immediately after the `if`